libnmstate/state.py

```diff
@@ -62,18 +112,17 @@ def __init__(self, route):
         self.complement_defaults()
 
     def complement_defaults(self):
-        if self.state != Route.STATE_ABSENT:
+        if not self.absent:
             if self.table_id is None:
                 self.table_id = Route.USE_DEFAULT_ROUTE_TABLE
             if self.metric is None:
                 self.metric = Route.USE_DEFAULT_METRIC
             if self.next_hop_address is None:
                 self.next_hop_address = ''
+            if self.destination is None:
```

Comment:
Somehow the review showed this difference now. (no idea why I did not noticed it earlier)
Does it make sense to complement this? How did it worked so far?

Reply:
I found that when coding. Yes. It should be in other PR.
We never test routes with destination(which is illegal), that's why it hides.